packages/data-driver-postgres/src/query/from.ts

```diff
@@ -0,0 +1,10 @@
+import { escapeIdentifier } from '../utils/escape-identifier.js';
+
+/**
+ * Generate the `FROM x` part of a SQL statement
+ * @param from - The table to select data from
+ * @returns The `FROM x` part of a SQL statement
+ */
+export function from(from: string): string {
```

Comment:
Lets use the query input here instead of a string so it's consistent with the other functions (like select/etc)